inc/vcf/error.hpp

```diff
@@ -172,6 +176,21 @@ namespace ebi
         virtual void apply_visitor(ErrorVisitor &visitor) override { visitor.visit(*this); }
     };
 
+    #pragma db object
+    struct NoMetaDefinitionError : public Error
+    {
+      private:
+        friend class odb::access;
+        NoMetaDefinitionError() {}
+      public:
+        NoMetaDefinitionError(size_t line, std::string message, std::string column, std::string field)
```

Comment:
Other classes in the hierarchy use `const std::string &` as string arguments for the constructor.